### PR TITLE
Added Packer and Vagrant files for the workbench base

### PR DESCRIPTION
**Pull request by vshatravenko**

### CODE REVIEW

File: templates/config/environments/%kite_env%/Vagrantfile.tt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Vagrant.configure("2") do |config|
```
Comment (mod) [inline templates/config/environments/%kite_env%/Vagrantfile.tt:1]: I wouldn't put Vagrant in env but at the base of Template.

File: templates/config/environments/%kite_env%/Vagrantfile.tt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  # SSH settings
+  config.ssh.username = 'kite'
+  config.ssh.password = ''
+
+  # Internal IP of the vm
+  config.vm.network :private_network, ip: "192.168.0.42"
+
+  # Shell script to provision the vm with
+  Vagrant.configure("2") do |config|
+    config.vm.provision "shell", privileged: true, path: "../../../scripts/provision.sh"
```
Comment (mod) [inline templates/config/environments/%kite_env%/Vagrantfile.tt:14]: try scripts/provision.sh

File: templates/config/environments/%kite_env%/Vagrantfile.tt [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  end
+
+  # Forwarded ports for Peatio and Barong
+  config.vm.network "forwarded_port", guest: 8000, host: 8000
+  config.vm.network "forwarded_port", guest: 8001, host: 8001
+
+  # Synced folders for Peatio and Barong
+  config.vm.synced_folder "peatio/", "/home/kite/peatio", owner: "kite", group: "users"
+  config.vm.synced_folder "barong/", "/home/kite/barong", owner: "kite", group: "users"
```
Comment (mod) [inline templates/config/environments/%kite_env%/Vagrantfile.tt:23]: need to think about this.
does it work ?

**Review by mod — CHANGES_REQUESTED** [review]
I would like just to move vagrant file

File: templates/config/environments/%kite_env%/Vagrantfile.tt (hunk 3 above)
Comment (vshatravenko) [reply templates/config/environments/%kite_env%/Vagrantfile.tt:23]: We should put symlinks in the repo and it should work fine

**Pull request merged**